hnswlib/hnswalg.h

```diff
@@ -32,7 +32,7 @@ namespace hnswlib {
             loadIndex(location, s);
         }
 
-        HierarchicalNSW(SpaceInterface<dist_t> *s, size_t max_elements, size_t M = 16, size_t ef_construction = 200) :
+        HierarchicalNSW(SpaceInterface<dist_t> *s, size_t max_elements, size_t M = 16, size_t ef_construction = 200, size_t random_seed = 0) :
```

Comment:
Can you please make random_seed = 100 by default? This was previously the default value.